core/data/src/main/java/uk/gov/gchq/gaffer/data/element/Edge.java

```diff
@@ -172,36 +191,55 @@ public String toString() {
                 + "} ";
     }
 
+    @JsonPOJOBuilder(withPrefix = "")
```

Comment:
Various Jackson annotations are required here to ensure that the Builder can be used to deserialise objects.